safedelete/managers.py

```diff
@@ -1,5 +1,6 @@
 from django.db import models
 
+from safedelete import config
```

Comment:
There already is a relative import for config, use that instead.

Reply:
Done.